cmd/fs-v1-metadata.go

```diff
@@ -375,6 +383,8 @@ func checkFormatSanityFSV2(fsPath string) error {
 // Check for sanity of FS format for a given version.
 func checkFormatSanityFS(fsPath string, fsFormatVersion string) (err error) {
 	switch fsFormatVersion {
+	case fsFormatV1:
```

Comment:
@harshavardhana  That's not how we had decided when we discussed with @abperiasamy 

what we decided was to do:

```
func initFormatFS() {
....
..
....
checkFormatSanityFSV5()
}
```

Here checkFormatSanityFSV5() will itself be no-op if it does not need to do anything. (But it might need to do sanity checks just like the current checkFormatSanityFSV2()) If it is a no-op then it will be a place holder so that when there is a version increment we know that we have to delete checkFormatSanityFSV5() and implement new checkFormatSanityFSV6()

There is no use of maintaining older sanity checking functions. For place holder we will just have the latest sanity checking function as no-op. (if it actually needs to do any sanity checks then it will not be a no-op)